docs/config_options.rst

```diff
@@ -3,43 +3,143 @@ Configuration Options
-
-- ``cache_sources``: If set to False, the default will be to not cache tile sources.  This has substantial performance penalties if sources are used multiple times, so should only be set in singular dynamic environments such as experimental notebooks.
-
-- ``max_small_image_size``: The PIL tilesource is used for small images if they are no more than this many pixels along their maximum dimension.
-
-- ``source_bioformats_ignored_names``, ``source_pil_ignored_names``, ``source_vips_ignored_names``: Some tile sources can read some files that are better read by other tilesources.  Since reading these files is suboptimal, these tile sources have a setting that, by default, ignores files without extensions or with particular extensions.  This setting is a Python regular expression.  For bioformats this defaults to ``r'(^[!.]*|\.(jpg|jpeg|jpe|png|tif|tiff|ndpi))$'``.
-
-- ``all_sources_ignored_names``: If a file matches the regular expression in this setting, it will only be opened by sources that explicitly match the extension or mimetype.  Some formats are composed of multiple files that can be read as either a small image or as a large image depending on the source; this prohibits all sources that don't explicitly support the format.
-
-- ``icc_correction``: If this is True or undefined, ICC color correction will be applied for tile sources that have ICC profile information.  If False, correction will not be applied.  If the style used to open a tilesource specifies ICC correction explicitly (on or off), then this setting is not used.  This may also be a string with one of the intents defined by the PIL.ImageCms.Intents enum.  ``True`` is the same as ``perceptual``.
-
-- ``max_annotation_input_file_length``: When an annotation file is uploaded through Girder, it is loaded into memory, validated, and then added to the database.  This is the maximum number of bytes that will be read directly.  Files larger than this are ignored.  If unspecified, this defaults to the larger of 1 GByte and 1/16th of the system virtual memory.
+.. list-table:: Configuration Parameters
+   :widths: 30 50 30 20 20
+   :header-rows: 1
+
+   * - Key(s)
+     - Description
+     - Type
+     - Required
+     - Default
+
+       .. _config_logger:
+   * - ``logger`` :ref:`🔗 <config_logger>`
+     - Most log messages are sent here.
+     - ``logging.Logger``
+     -
+     -
+
+       .. _config_logprint:
+   * - ``logprint`` :ref:`🔗 <config_logprint>`
+     - Messages about available tilesources are sent here.
+     - ``logging.Logger``
+     -
+     -
+
+       .. _config_cache_backend:
+   * - ``cache_backend`` :ref:`🔗 <config_cache_backend>`
+     - String specifying how tiles are cached.  If memcached is not available for any reason, the python cache is used instead.
+     - ``str: "python" | "memcached"``  # What about redis?
+     - ``False``
+     - ``"python"``
```

Comment:
This can also be ``None``, which is the default.  When ``None``, the first cache available in the order memcached, redis, python is used.  Otherwise, the specified cache is used if available, falling back to python if not.